{"name": "test_login_with_valid_email_and_invalid_password", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = invalid_cred_warning_message('Warning: No match+++ for E-Mail Address and/or Password.')\n +    where invalid_cred_warning_message = <pom.LoginPage.LoginPage object at 0x00000250B4737BB0>.invalid_cred_warning_message", "trace": "self = <tests.test_login.TestLogin object at 0x00000250B46D4050>\n\n    def test_login_with_valid_email_and_invalid_password(self):\n        home_page = HomePage(self.driver)\n        login_page=home_page.reach_to_login_page()\n        time.sleep(1)\n        expected_warning_msg = \"Warning: No match+++ for E-Mail Address and/or Password.\"\n        login_page.user_login_process_invalid(\"123456\",expected_warning_msg)\n        time.sleep(1)\n>       assert login_page.invalid_cred_warning_message(expected_warning_msg)\nE       AssertionError: assert False\nE        +  where False = invalid_cred_warning_message('Warning: No match+++ for E-Mail Address and/or Password.')\nE        +    where invalid_cred_warning_message = <pom.LoginPage.LoginPage object at 0x00000250B4737BB0>.invalid_cred_warning_message\n\ntests\\test_login.py:28: AssertionError"}, "start": 1759059640772, "stop": 1759059644748, "uuid": "55ceefaf-307f-40fc-bf9a-86de91f7956d", "historyId": "a87129e8bc488f2c2b04474f72c047f1", "testCaseId": "a87129e8bc488f2c2b04474f72c047f1", "fullName": "tests.test_login.TestLogin#test_login_with_valid_email_and_invalid_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-PQJ2IUR"}, {"name": "thread", "value": "26300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py", "TestLogin"]}